pom.xml

```diff
@@ -30,7 +30,10 @@
     <coffee-boots.version>4.0.0</coffee-boots.version>
     <snakeyaml.version>2.0</snakeyaml.version>
     <org.postgresql.version>42.5.4</org.postgresql.version>
-    <org.jooq.version>3.18.4</org.jooq.version>
+    <!-- [2025-11-26] 3.19.21 onwards causes issues with Record1::value1 in IdStreamer -->
+    <!-- It appears these return a String[][] when they should (and declare) String[] -->
+    <!-- Attempting to use it in our logic (to make List<String>) results in casting exceptions -->
+    <org.jooq.version>3.19.20</org.jooq.version>
```

Comment:
We should use a newer `3.20.x` version, but it results in weird errors that seem like a broken API in jOOQ. However, this is out of the scope of this (already large ticket), so I'm saving that for later. We do need 3.19.0+, though, for the `dropMaterializedView` method.

From what I can tell, `Record1::value1` is returning a `String[][]` with an extra layer of wrapping compared to what it should be (and what it is declared as).

<img width="463" height="196" alt="Image" src="https://github.com/user-attachments/assets/fbd155b8-d7c1-43d9-a849-144badb93ceb" />